src/main/definitions/radios.ts

```diff
@@ -46,7 +46,7 @@ export const SupportingMaterialYesNoRadioValues = [
 ];
 
 export const submitButton: SubmitButton = {
-  text: (l: AnyRecord): string => l.submit,
+  text: (l: AnyRecord): string => l.saveAndContinue,
```

Comment:
the method is submitButton but label saveAndContinue. is this a little confusing